bukuserver/views.py

```diff
@@ -425,28 +415,32 @@ class TagModelView(BaseModelView, ApplyFiltersMixin):
     def _create_ajax_loader(self, name, options):
         pass
 
-    def _name_formatter(self, _, model, name):
+    def _name_formatter(self, context, model, name):
         data = getattr(model, name)
         query, title = (({'flt0_tags_contain': data}, data) if data else
-                        ({'flt0_tags_number_equal': 0}, '<UNTAGGED>'))
-        return Markup(link(title, url_for("bookmark.index_view", **query)))
+                        ({'flt0_tags_number_equal': 0}, _('<UNTAGGED>')))
+        return Markup(link(title, url_for('bookmark.index_view', **query)))
 
     can_create = False
     can_set_page_size = True
-    column_filters = ["name", "usage_count"]
-    column_formatters = {
-        "name": _name_formatter,
-    }
+    column_filters = ['name', 'usage_count']
+    column_labels = {'name': _lp('tag', 'Name'), 'usage_count': _lp('tag', 'Usage Count')}
```

Comment:
Using namespaced strings here to allow for tag-specific translations of these labels